compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/TypeChecker.java

```diff
@@ -8634,8 +8634,8 @@ private BType checkFieldAccessExpr(BLangFieldBasedAccess fieldAccessExpr, BType
             fieldAccessExpr.originalType = actualType;
             if (actualType == symTable.semanticError) {
                 dlog.error(fieldAccessExpr.pos, DiagnosticErrorCode.UNDEFINED_STRUCTURE_FIELD_WITH_TYPE,
-                        fieldName,
-                        varRefType.getKind() == TypeKind.UNION ? "union" : varRefType.getKind().typeName(), varRefType);
+                        fieldName, varRefType.getKind() == TypeKind.UNION ? "union" : varRefType.getKind().typeName(),
```

Comment:
This is just a formatting change compared to previous version right? How does it fix the original issue?